src/command_modules/azure-cli-container/azure/cli/command_modules/container/_params.py

```diff
@@ -65,7 +65,12 @@ def load_arguments(self, _):
         c.argument('azure_file_volume_share_name', help='The name of the Azure File share to be mounted as a volume')
         c.argument('azure_file_volume_account_name', help='The name of the storage account that contains the Azure File share')
         c.argument('azure_file_volume_account_key', help='The storage account access key used to access the Azure File share')
-        c.argument('azure_file_volume_mount_path', validator=validate_volume_mount_path, help='The path within the container where the volume should be mounted. Must not contain colon (:).')
+        c.argument('azure_file_volume_mount_path', validator=validate_volume_mount_path, help='The path within the container where the azure file volume should be mounted. Must not contain colon (:).')
+
+    with self.argument_context('container create', arg_group='Git Repo Volume') as c:
+        c.argument('gitrepo_repository', help='The URL of a git repository to be mounted as a volume')
+        c.argument('gitrepo_directory', validator=validate_gitrepo_directory, help="The target directory path in the git repository. Must not contain '..'.")
+        c.argument('gitrepo_mount_path', validator=validate_volume_mount_path, help='The path within the container where the git repo volume should be mounted. Must not contain colon (:).')
```

Comment:
A small detail but for `gitrepo_directory ` you use `''` around `..` for 'must not contain' but for `gitrepo_mount_path` you use `()` around `:`.
For consistency just use one or the other IMO.